docs/scenarios/parametric.md

```diff
@@ -97,9 +92,36 @@ TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -k test_metrics_
 
 Tests can be aborted using CTRL-C but note that containers maybe still be running and will have to be shut down.
 
-#### Go
+### Using Pytest
+
+The tests are executed using pytest. Below are some common command-line options you can use to control and customize your test runs.
+- `-k EXPRESSION`: Run tests that match the given expression (substring or pattern). Useful for running specific tests or groups of tests.
+
+```sh
+TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -k test_metrics_msgpack_serialization_TS001
+```
+
+- `-v`: Increase verbosity. Shows each test name and its result (pass/fail) as they are run.
+
+```sh
+TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -v
+```
+
+- `-vv`: Even more verbose output. Provides detailed information including setup and teardown for each test.
+
+```sh
+TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -vv -k test_metrics_
+```
+
+- `-s`: Disable output capture. Allows you to see print statements and logs directly in the console.
+
+```sh
+TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -s
+```
+
+### Running the tests for a custom tracer
```

Comment:
So we have this binaries.md file which talks about how to build custom tracers for each language (learned about this recently): https://github.com/DataDog/system-tests/blob/main/docs/execute/binaries.md?plain=1

Could we modify the necessary pieces of binaries.md for each tracer (basically just adding anything that we have in parametric.md regarding this) and then link to binaries.md sections for each custom tracer part? That way the information is just kept in a single place.